frontend/components/utils/TimeTillProfitable.stories.js

```diff
@@ -0,0 +1,19 @@
+import { storiesOf } from '@storybook/vue';
+import TimeTillProfitable from '~/components/utils/TimeTillProfitable';
+import { generateFakeAuction } from '~/helpers/generateFakeAuction';
+
+const auction = generateFakeAuction();
+
+const common = {
+    components: { TimeTillProfitable },
+    data() {
+        return {
+            auction,
+        };
+    },
+};
+
+storiesOf('Utils/TimeTillProfitable', module).add('Default', () => ({
+    ...common,
+    template: '<TimeTillProfitable :auction="auction" />',
+}));
```

Comment:
Can you please cover all edge cases via stories?
- inactive / finished auctions
- never profitable
- already profitable
- etc

Reply:
finished auction is not yet covered!